Makefile

```diff
@@ -0,0 +1,74 @@
+
+.PHONY: utility fetch-engine build build-engine clean clean-engine version check-scripts check test
+.DEFAULT_GOAL := build
+
+VERSION = $(shell git name-rev --name-only --tags --no-undefined HEAD 2>/dev/null || echo git-`git rev-parse --short HEAD`)
+MOD_ID = $(shell awk -F= '/MOD_ID/ { print $2 }' mod.config | cut -d'=' -f 2)
+INCLUDE_DEFAULT_MODS = $(shell awk -F= '/INCLUDE_DEFAULT_MODS/ { print $2 }' mod.config | cut -d'=' -f 2)
+
+MOD_SEARCH_PATHS = "$(shell pwd)/mods"
+ifeq ("$(INCLUDE_DEFAULT_MODS)","True")
+	MOD_SEARCH_PATHS = "$(MOD_SEARCH_PATHS),./mods"
+endif
+
+MANIFEST_PATH = "mods/$(MOD_ID)/mod.yaml"
+
+HAS_MSBUILD = $(shell command -v msbuild 2> /dev/null)
+
+LUA_FILES = $(shell find mods/*/maps/* -iname '*.lua')
+
+utility:
+	@test -f engine/OpenRA.Utility.exe || printf "OpenRA.Utility.exe not found! Build the engine first.\n"
+
+fetch-engine:
+	@test -f engine/OpenRA.sln || git submodule update --init
+
+build: build-engine
+ifeq ("$(HAS_MSBUILD)","")
+	@xbuild /nologo /verbosity:quiet /p:TreatWarningsAsErrors=true
+else
+	@msbuild /t:Rebuild /nr:false
```

Comment:
Please update the Copy tasks in the .csproj to:
```
<Copy SourceFiles="$(TargetDir)$(TargetName).pdb" DestinationFolder="$(SolutionDir)mods/example/" Condition="Exists('$(TargetDir)$(TargetName).pdb')" />
<Copy SourceFiles="$(TargetPath).mdb" DestinationFolder="$(SolutionDir)mods/example/" Condition="Exists('$(TargetPath).mdb')" />
```

for compatibility with msbuild.  I will file a pr against OpenRA/OpenRA to fix our other csprojes, which is needed for build-engine to work.